find_clang.cr, function parse_clang_output takes only first include path while skipping others

For example I have this flags (https://github.com/Papierkorb/bindgen/blob/master/clang/find_clang.cr#L380), I'll include only `-internal-isystem` part:
```
...
 "-internal-isystem",
 "/usr/bin/../lib/gcc/x86_64-redhat-linux/10/../../../../include/c++/10",
 "/usr/bin/../lib/gcc/x86_64-redhat-linux/10/../../../../include/c++/10/x86_64-redhat-linux",
 "/usr/bin/../lib/gcc/x86_64-redhat-linux/10/../../../../include/c++/10/backward",
 "/usr/local/include",
 "/usr/lib64/clang/10.0.0/include",
 "-internal-externc-isystem",
...
```

And this function returns only first one dir - `/usr/include/c++/10`, skipping the `/usr/local/include`, `/usr/lib64/clang/10.0.0/include` and others.
And because I don't have `/usr/lib64/clang/10.0.0/include` in include paths defined, later I see this error running `crystal spec`:
```
In file included from /tmp/.kMdBOGbindgen:1:
In file included from /home/zawertun/tmp/bindgen@Papierkorb/spec/integration/c_wrapper.cpp:1:
In file included from /usr/lib/gcc/x86_64-redhat-linux/10/../../../../include/c++/10/cstdlib:75:
/usr/include/stdlib.h:31:10: fatal error: 'stddef.h' file not found
#include <stddef.h>
         ^~~~~~~~~~
```
File `stddef.h` can be found in the `/usr/lib64/clang/10.0.0/include` dir, if I add this include manually - `crystal spec` finishes without errors.

--- a/clang/find_clang.cr
+++ b/clang/find_clang.cr
@@ -379,11 +379,22 @@ def parse_clang_output(output)
   # Interpret the argument lists
   flags = cppflags + ldflags
   index = 0
+  internal_isystem = false
   while index < flags.size
+    flag = flags[index]
+    if internal_isystem
+      if flag[0] == '-'
+        internal_isystem = false
+      else
+        system_include_dirs << flag
+        index += 1
+        next
+      end
+    end
+
     case flags[index]
     when "-internal-isystem"
-      system_include_dirs << flags[index + 1]
-      index += 1
+      internal_isystem = true
     when "-resource-dir" # Find paths on Ubuntu
       resource_dir = flags[index + 1]
       system_include_dirs << File.expand_path("#{resource_dir}/../../../include")

